src/Generator/Template/ParameterTemplate.php

```
@@ -32,7 +32,7 @@ protected static function doGenerate(\ReflectionParameter $parameter): string
                 : 'null';
 
             $defaultValue = '=' . $defaultValue;
-        } catch (\ReflectionException $exception) {
+        } catch (\Exception $exception) {
```

Is it really safe silencing all the exceptions?